.env

```diff
@@ -0,0 +1 @@
+USER_PASSWORD=plingplong
```

Comment:
Secrets should not be public! If this was a real application you would be in trouble for this